draft-ietf-quic-ack-frequency.md

```diff
@@ -106,15 +106,15 @@ Working Group information can be found at [](https://github.com/quicwg).
 
 # Introduction
 
-The QUIC transport protocol recommends to send an ACK frame
-after receiving at least two ack-eliciting packets; see
-{{Section 13.2 of QUIC-TRANSPORT}}. However, it leaves the determination
-of how frequently to send acknowledgments in response to ack-eliciting packets
-to the data receiver without any ability for the data sender to impact this
-behavior. This document specifies an extension to QUIC that permits an endpoint to
-request that its peer changes its behavior when sending or delaying acknowledgments.
-
-This document defines a new transport parameter to announce the support of this
+The QUIC transport protocol recommends sending an ACK frame after receiving at
+least two ack-eliciting packets; see {{Section 13.2 of QUIC-TRANSPORT}}.
+However, this protocol leaves the data sender unable to indicate how frequently
+it would prefer the data receiver send acknowledgements in response to
+ack-eliciting packets. This document specifies an extension to QUIC enabling
```

Comment:
```suggestion
ack-eliciting packets. This document specifies an extension to QUIC that enables
```
This sentence is the same as in the abstract.

Reply:
Agreed, should we remove it?

Reply:
No, we need to say this in the body as well (and I think it is actually good practice to repeat the same wording). See the RFC style guide section 4.3 (https://www.rfc-editor.org/rfc/rfc7322.html#section-4.3):

>   Note
>   also that an Abstract is not a substitute for an Introduction; the
>   RFC should be self-contained as if there were no Abstract.

Reply:
Thanks, suggestion applied.